pyhf/commandline.py

```diff
@@ -44,13 +44,14 @@ def json2xml(workspace,xmlfile,specroot,dataroot):
 @pyhf.command()
 @click.argument('workspace', default = '-')
 @click.option('--output-file', help='The location of the output json file. If not specified, prints to screen.', default=None)
-def cls(workspace,output_file):
+@click.option('--qualify-names/--no-qualify-names', default=False)
+def cls(workspace, output_file, qualify_names):
     specstream = click.open_file(workspace)
     d = json.load(specstream)
     measurements = d['toplvl']['measurements']
     if len(measurements) > 1:
```

Comment:
maybe we can actually have this as a similar flag to specify which of the defined measurements in the workspace to fit? `--measurement`

Reply:
done

Reply:
See #234.